backend/src/main/java/com/woowacourse/gongcheck/domain/submission/SubmissionRepository.java

```diff
@@ -1,6 +1,15 @@
 package com.woowacourse.gongcheck.domain.submission;
 
+import com.woowacourse.gongcheck.domain.job.Job;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface SubmissionRepository extends JpaRepository<Submission, Long> {
+
+    @Query(value = "SELECT sub FROM Submission sub JOIN FETCH sub.job WHERE sub.job in :jobs")
+    Slice<Submission> findAllByJobIn(@Param("jobs") final List<Job> jobs, Pageable pageable);
```

Comment:
```suggestion
    Slice<Submission> findAllByJobIn(@Param("jobs") final List<Job> jobs, final Pageable pageable);
```
일관성이긴한데 따로 인터페이스에 있던 final들을 일괄적으로 리팩터링해야겠네요 ㅎㅎ...